pkg/k8shandler/certificates.go

```diff
@@ -27,37 +26,39 @@ func extractMasterCertificate(namespace string, secretName string)(error) {
   }
   err := sdk.Get(secret)
 
-  // value []byte
-  caValue, ok := secret.Data["masterca"]
+  value, ok := secret.Data[key]
 
   // check to see if the map value exists
   if ! ok {
-    logrus.Fatalf("No secret data \"masterca\" found")
+    logrus.Fatalf("No secret data \"%s\" found", key)
     return nil
   }
 
-  err = ioutil.WriteFile("/tmp/_working_dir/ca.crt", caValue, 0644)
+  err = ioutil.WriteFile("/tmp/_working_dir/" + toFile, value, 0644)
```

Comment:
* refactor `/tmp/_working_dir/` to a constant.  
* Consider using [path](https://golang.org/pkg/path/) lib to join file name and parent